plugins/tileacq.py

```diff
@@ -514,7 +544,13 @@ def estimate_time(self, remaining=None):
 
         if remaining is None:
             remaining = self.nx.value * self.ny.value
-        acqt = acq.estimateTime(ss)
+
+        # Add the overlay stream if requested
+        if self.fine_align.value and self._can_fine_align(self._get_live_streams()):
```

Comment:
I think you could just move this code to `_get_acq_streams()` so that it returns also the `ovrl_stream` if requested.

Reply:
I tried this at first, but the problem is that later we use all streams in `_get_acq_streams()` for stitching, so that's not going to work with the overlay.

Reply:
Then you could do the same as we did in the timelapse plugin: `_get_acq_streams()` returns two lists. Here it could either:
 * the list of all the streams, and list with only the streams useful for stitching
 * the list with only the streams useful for stitching, and the list of "extra streams"